calibrators/intrinsic_camera_calibrator/ceres_intrinsic_camera_calibrator/src/ceres_intrinsic_camera_calibrator_py.cpp

```diff
@@ -182,14 +184,16 @@ PYBIND11_MODULE(ceres_intrinsic_camera_calibrator_py, m)
             num_radial_coeffs (int): The number of radial distortion coefficients used during calibration
             num_rational_coeffs (int): The number of rational distortion coefficients used during calibration
             use_tangential_distortion (bool): Whether we should use tangential distortion during calibration
+            regularization_weight (float): The regularization weight for distortion coefficients
```

Comment:
was it not set to double?

Reply:
Addressed in https://github.com/tier4/CalibrationTools/pull/208/commits/ca7b41c45b9f5dc21ce09fc17fa98bb0643154b6